python/lsst/obs/pfs/createDetGeom.py

```diff
@@ -0,0 +1,144 @@
+# example: createDetGeom.py
+import os
+import lsst.pex.config                  as pexConfig
+import lsst.afw.table as afwTable
+import lsst.afw.geom as afwGeom
+from lsst.pipe.base import Task
+import lsst.utils
+
+class CreateDetGeomConfig(pexConfig.Config):
+    """Configuration for creating the PFS detector geometry files"""
+
+class CreateDetGeomTask(Task):
+    """Task to create the detector geometry files for PFS"""
+    ConfigClass = CreateDetGeomConfig
+    _DefaultName = "createDetGeomTask"
+
+    def __init__(self, *args, **kwargs):
+        Task.__init__(self, *args, **kwargs)
+
+    def run(self):
+        
+        """ Gain and RdNoise from Jim Gunn's talk in Marseille Dec 2015"""
+        gain = (1.24, 1.24, 1.27, 1.18, 1.26, 1.20, 1.24, 1.26)
+        rdnoise = (3.61, 3.78, 3.18, 2.95, 3.19, 3.80, 4.51, 3.18)
+        
+        # According to RHL's data model (https://github.com/Subaru-PFS/datamodel/blob/master/datamodel.txt):
+        # The raw CCD images will have:
+        #    NAMP=8
+        #    AMPCOLS=520
+        #    CCDROWS=4224
+        #
+        #    OVERROWS = 76 # as of 2016-02-01
+        #    OVERCOLS = 32 # as of 2016-02-01
+        #
+        #    LEADINROWS = 48 # necked rows
+        #    LEADINCOLS  = 8 # real leadin pixels
+        #
+        # So the raw CCD data will have
+        #    NROW = 4300 #  CCDROWS + OVERROWS
+        #    NCOL = 4416 # NAMP*(AMPCOLS + OVERCOLS)
+        #
+        # And the ISR-extracted CCD images will probably be close to:
+        #    NROW = 4172  # CCDROWS - LEADINROWS
+        #    NCOL = 4096  # NAMP*(AMPCOLS - LEADINCOLS)
+        # This value of NROW is applicable to the pfsArm files.
+
+        #                        according to Craig Loomis:
+        nColsAmp = 512#          512
+        nColsHPrescan = 8#9#     8 + 1 for first, then 8
```

Comment:
We don't yet, still discussing what to do about these files, fix them or use a different detector geometry. See SCAM-5